src/Smp.Web/ClientApp/package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "echo something && ng serve",
```

Comment:
This is a temporary things because without it, the .NET Core project doesn't thing Angular is ready. This might have something to do with Angular 9 printing a different message with serve than before.